applications/vessel-history/src/features/map/Map.tsx

```diff
@@ -120,7 +136,7 @@ const Map = (): ReactElement => {
           essential: true, // this animation is considered essential with respect to prefers-reduced-motion
         })
         mapRef?.current?.getMap().fire('flystart')
-        dispatchQueryParams({ latitude: nextEvent.position.lat, longitude: nextEvent.position.lon })
+        //dispatchQueryParams({ latitude: nextEvent.position.lat, longitude: nextEvent.position.lon })
```

Comment:
leave this comment, I want to remember something

Reply:
let's add a comment with the reason you're leaving it then 😉

Reply:
personal issues haha, no because looks like is not necessary but I remember that I added because a problem when the map arrives to the destination and try to manipulate the map, I didn't see again the issue but if appear I remember that this is the fix, if nothing happen I will remove soon